modules/installation-azure-limits.adoc

```diff
@@ -184,6 +184,17 @@ ifndef::ash[]
 endif::ash[]
 |The internal load balancer, each of the three control plane machines, and each
 of the three worker machines each use a private IP address.
+
+ifndef::ash[]
+|Spot VMs
+|0
+|20 per region
+|This is an optional component. To use spot VMs, you must increase the spot VM limits.
```

Comment:
Also here correct it to "the spot VM vCPU limits"

Reply:
Commend addressed.